src/cleed++/Core/MillerIndex.cc

```
@@ -0,0 +1,68 @@
+#include <sstream>
+
+#include "MillerIndex.hh"
+
+using namespace cleed;
+
+MillerIndex::MillerIndex(
+  double h, 
```

Clang (14.0.6)

[warning] 3 adjacent parameters of 'MillerIndex' of similar type ('double') are easily swapped by mistake

(bugprone-easily-swappable-parameters)